### PR TITLE
react-textarea: Re-exporting Textarea in react-components

### PR DESCRIPTION
**Pull request by sopranopillow**
## PR details

This PR introduces `react-textarea` to `react-components`, adds more detail to the readme, and adds a label to the default example.

## Related issues
Issues: #21845 and #22320

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `176.089 kB`<br />`49.113 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.601 kB`<br />`10.668 kB` |
| <samp>react-textarea</samp> <br /> <abbr title='bundle-size/Textarea.fixture.js'>Textarea</abbr>                                                                              |   `20.602 kB`<br />`7.033 kB` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/126d1921f2a886bbb96fd1e94d542255024c91b7'>126d1921f2a886bbb96fd1e94d542255024c91b7</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/Avatar-mount_base.html">928</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/Avatar-mount.html">919</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/Button-mount_base.html">541</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/Button-mount.html">538</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProvider-mount_base.html">1971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProvider-mount.html">1879</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProviderWithTheme-mount_base.html">268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProviderWithTheme-mount.html">253</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">244</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">287</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">300</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/MakeStyles-mount_base.html">1567</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22413/perf-test-react-components/MakeStyles-mount.html">1556</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Review by smhigley — APPROVED** [review]

File: packages/react-textarea/package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@fluentui/react-textarea",
   "version": "9.0.0-alpha.0",
-  "private": true,
+  "private": false,
```
Comment (smhigley) [inline packages/react-textarea/package.json:4]: I think you can just remove this line entirely

**Pull request merged**